### DISCUSSION TITLE
Bug in RTL Dashboard??

### DISCUSSION THREAD

**Discussion by shobitb** [Ride-The-Lightning/RTL#1266] [Q&A] [answered]
**Describe the bug**
I had a channel open with "ACINQ" that got disconnected/closed (not sure why), but I did not receive the sats back (or so the RTL Dashboard says). I don't know what to make out of it. Is this a display bug, or did I lose the funds?

**Screenshots**
<img width="751" alt="Screen Shot 2023-06-29 at 9 51 50 AM" src="https://github.com/Ride-The-Lightning/RTL/assets/1744097/097694ea-2fe0-4c9b-90c2-48140675b861">
--

<img width="928" alt="Screen Shot 2023-06-29 at 9 52 04 AM" src="https://github.com/Ride-The-Lightning/RTL/assets/1744097/6b201660-3f00-4779-aeab-ed30cc6ef8bd">

**Your environment**
* Umbrel - umbrelOS Version 0.5.4
* Version of `RTL` - 0.13.2-beta

**Comment by saubyk** [comment, marked as answer]
Hi @shobitb can you clarify what you mean by "I did not receive the sats back"?
Ideally, if the channel is closed or force closed your funds will be back on chain. So you should check your on chain balance instead.

The screenshots that you attached is only showing lightning balance, which would be nil since your only channel has been closed.

tracking the onchain activity from the channel funding transaction above, you should get 1,192,838 sats back in your wallet
https://mempool.space/address/bc1q7qjnpsk7lxcqx4s9e2473vkec3lv2c26j4s07mxzhp0786qk7fjqwa5ace

**Reply by shobitb** [reply]
I appreciate your quick response, @saubyk .  What I am trying to understand is why the RTL Dashboard does not show the on-chain sats, or rather shows it as 0 sats on-chain.  

Also, when you say it should be back in my wallet, is this something I can check on RTL Dashboard?  Which wallet?


<img width="1425" alt="Screen Shot 2023-06-29 at 10 07 27 AM" src="https://github.com/Ride-The-Lightning/RTL/assets/1744097/cb40f831-9f95-4ec6-b40a-9a3f511d4449">

**Reply by saubyk** [reply]
Looks like your transaction is still unconfirmed:
https://mempool.space/tx/1e225a531a90365f757cec48c87ec055fc50cc422a8f97f7fbd69a2b046099d9

Which implementation are you on? My suspicion is core lightning, which doesn't show unconfirmed on chain balance

You can try to do CPFP, to bump up your pending transaction, but that would require accessing the command line interface on Umbrel

**Reply by shobitb** [reply]
Oh! That makes sense. Yes, using Core Lightning. 

I will figure out how to do CPFP via command line.

Thanks for your help!

**Reply by saubyk** [reply]
​​When you attempt CPFP, keep these things in mind:
- It appears that your transaction is more than 2 weeks old, which implies that it might have been dropped from most mempools
- If the CPFP fails in your first attempt, you might need to rebroadcast the original transaction (I am not sure how that would work given that this is a channel close, you may need to research a bit, or reach out to core lightning team for support)
- Keep the prevailing fee rates in mind when you try to CPFP, your child transaction should sufficiently bid over the top rate so that the average rate works out to be good enough for the miners

Hope this helps
